assets/charts/components/cluster-autoscaler/templates/secret.yaml

```diff
@@ -16,21 +17,24 @@ data:
 {{- else if eq .Values.cloudProvider "aws" }}
   AwsAccessKeyId: "{{ .Values.awsAccessKeyID | b64enc }}"
   AwsSecretAccessKey: "{{ .Values.awsSecretAccessKey | b64enc }}"
+# XXX: Lokomotive specific change.
 {{- else if eq .Values.cloudProvider "packet" }}
 type: Opaque
 stringData:
   cloud-config: |-
     [Global]
-    project-id= {{ .Values.packetProjectID }}
-    facility= {{ .Values.packetFacility }}
-    hostname-pattern={{ printf "%s" "{{.ClusterName}}-{{.NodeGroup}}-{{.RandString8}}" }}
-    os=flatcar_{{ .Values.packetOSChannel }}
+    project-id={{ .Values.packetProjectID }}
+    api-server-endpoint={{ .Values.apiServer }}
```

Comment:
What does this `api-server-endpoint` change do ?

can we minimize the order changes ?

Reply:
I think I took it from [upstream](https://github.com/kubernetes/autoscaler/blob/5766bb71373a8f94faa0b8458b747cfbb89c1344/cluster-autoscaler/cloudprovider/packet/examples/cluster-autoscaler-secret.yaml#L25). I was trying to use the upstream configs to make this work with packet.